doc/nrf/libraries/zigbee/lib_zigbee_signal_handler.rst

```diff
@@ -275,6 +275,11 @@ The minimal inactivity duration that causes the signal to be generated is define
 By default, the inactivity duration equals approximately 15 ms.
 The value can be modified by the ``zb_sleep_set_threshold`` API.
 
+    .. figure:: /images/zigbee_signal_handler_07_idle.png
+       :alt: Generation of the ZB_COMMON_SIGNAL_CAN_SLEEP signal
+
+       Generation of the ZB_COMMON_SIGNAL_CAN_SLEEP signal
+
```

Comment:
This needs to be aligned with the text. Else, it's good to go.

Reply:
Fixed.